server/server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const io = require('socket.io-client');
+const socket = io.connect('http://localhost:3000');
```

Comment:
Try this `const io = require('socket.io')(3000);` instead of lines 2 and 3.  the replace `socket` with `io` on line 6.